### PR TITLE
Added customizable asset picker component docs

### PR DESCRIPTION
**Pull request by nitinja**
Added customizable asset picker component docs
## Related Issue

https://jira.corp.adobe.com/browse/SITES-19685

### CODE REVIEW

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+Following type of configuration is available:
+- AEM tier type ("delivery", "author" etc.)
+- Repo names to select assets from
+- ApiKey
+
+Configuration also supports following types of filters:
+- File type (Images, Video etc.)
+- Asset size (height, width)
+
+## Configuration in Crosswalk Site
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:27]: I would call it Edge Delivery Site.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+---
+title: Configurable Asset Picker
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:2]: Universal Editor Custom Asset Picker

**Comment by fe-lix-** [comment]
We should also have a section with current limitation.

We are limited to a single custom component: https://git.corp.adobe.com/dx-devex-acceleration/uix-examples-internal/blob/main/uex-asset-picker/src/aem-uex-1/web-src/src/components/ExtensionRegistration.js#L33

We cannot open open the custom asset picker from the remote app view. As in we cannot double click on the image in the wysiwyg and open the custom asset picker.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md (hunk 2 above)
Comment (nitinja) [reply src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md]: I was under impression that we agreed upon "Configurable asset picker?". Anyway, I will change name everywhere in this docs.

Comment (fe-lix-) [reply src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md]: Sorry.
It was the Universal Editor I wanted to emphasize.

**Comment by nitinja** [comment]
> We should also have a section with current limitation.
> 
> We are limited to a single custom component: https://git.corp.adobe.com/dx-devex-acceleration/uix-examples-internal/blob/main/uex-asset-picker/src/aem-uex-1/web-src/src/components/ExtensionRegistration.js#L33
> 
> We cannot open open the custom asset picker from the remote app view. As in we cannot double click on the image in the wysiwyg and open the custom asset picker.

- Added limitations section
- Limited to single ctom component: this does not limit ability to crete many variations of asset picker by providing different configurations, so I think practically it won't matter. We can discuss and then add this.

File: src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+                            "label": "Image",
+                            "value": "image/*",
+                            "readOnly": true
+                        }
+                    ],
+                    "orientation": "horizontal"
+                }
+            ],
+            "header": "File Type",
+            "groupKey": "FileTypeGroup"
```
Comment (fe-lix-) [inline src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md:171]: We should add a default schema.

**Pull request merged**